_sass/minima/_layout.scss

```diff
@@ -25,77 +25,100 @@
 }
 
 .site-nav {
-  float: right;
+  position: absolute;
+  top: 9px;
+  right: $spacing-unit / 2;
+  background-color: $background-color;
+  border: 1px solid $grey-color-light;
+  border-radius: 5px;
+  text-align: right;
   line-height: $base-line-height * $base-font-size * 2.25;
 
   .nav-trigger {
       display: none;
   }
 
   .menu-icon {
+    float: right;
+    width: 36px;
+    height: 26px;
+    line-height: 0;
+    padding-top: 10px;
+    text-align: center;
+
+    > svg path {
+      fill: $grey-color-dark;
+    }
+  }
+
+  label[for="nav-trigger"] {
+    display: block;
+    float: right;
+    width: 36px;
+    height: 36px;
+    z-index: 2;
+    cursor: pointer;
+  }
+
+  input ~ .trigger {
+    clear: both;
     display: none;
   }
 
+  input:checked ~ .trigger {
+    display: block;
+    padding-bottom: 5px;
+  }
+
   .page-link {
     color: $text-color;
     line-height: $base-line-height;
+    display: block;
+    padding: 5px 10px;
 
     // Gaps between nav items, but not on the last one
     &:not(:last-child) {
-      margin-right: 20px;
+      margin-right: 0;
     }
+    margin-left: 20px;
   }
 
-  @include media-query($on-palm) {
-    position: absolute;
-    top: 9px;
-    right: $spacing-unit / 2;
-    background-color: $background-color;
-    border: 1px solid $grey-color-light;
-    border-radius: 5px;
-    text-align: right;
+  @include media-query($on-medium) {
+    position: static;
+    float: right;
+    border: none;
+    background-color: inherit;
 
     label[for="nav-trigger"] {
-      display: block;
+      display: inline;
       float: right;
-      width: 36px;
-      height: 36px;
+      width: auto;
+      height: auto;
       z-index: 2;
       cursor: pointer;
     }
```

Comment:
Ah, yes. And so, I'm removing all styles for `label [for="nav-trigger"]` , except for `display` which should be `display:none` like `.menu-icon` @ `$on-medium`